schemachange/sc_callbacks.c

```diff
@@ -186,6 +186,7 @@ unsigned long long get_genid_stripe_pointer(unsigned long long genid,
                                             unsigned long long *sc_genids)
 {
     int stripe = get_dtafile_from_genid(genid);
+    // TODO: can this ever be out of range?
```

Comment:
It appears that get_dtafile_from_genid() can return a negative value if the genid is synthetic.  Using that negative value here would indeed be bad, given that it's used as an array index.  Related: since gbl_dtastripe is based on a tunable, I wonder how it is validated?

Reply:
gbl_dtastripe should be a readonly tunable - it's set at db create time.  There's a 'morestripe' command to expand it.  It's rarely done.